packages/engine/src/renderer/functions/configureEffectComposer.ts

```diff
@@ -45,10 +44,14 @@ import { SceneState } from '../../ecs/classes/Scene'
 import { getComponent } from '../../ecs/functions/ComponentFunctions'
 import { PostProcessingComponent } from '../../scene/components/PostProcessingComponent'
 import { EffectMap, EffectPropsSchema, Effects } from '../../scene/constants/PostProcessing'
-import { RendererState } from '../RendererState'
-import { EffectComposerWithSchema, EngineRenderer, PostProcessingSettingsState } from '../WebGLRendererSystem'
+import { EffectComposerWithSchema, EngineRenderer } from '../WebGLRendererSystem'
 import { changeRenderMode } from './changeRenderMode'
 
+// can use both as needed
+export const debouncedConfigureEffectComposer = debounce(() => {
```

Comment:
we can keep the debounce function in the editor module.

Reply:
Editor.ts?

Reply:
if you mean as a dependency in package.json, its already there

Reply:
in PostProcessingSettingsEditor.tsx